src/pages/PoolDetails/PoolDetailsStats.test.tsx

```diff
@@ -11,8 +11,11 @@ describe('PoolDetailsStats', () => {
     chainId: 1,
   }
 
-  it('renders stats text correctly', () => {
+  it('renders stats text correctly', async () => {
     const { asFragment } = render(<PoolDetailsStats {...mockProps} />)
+    await act(async () => {
```

Comment:
this is a strange enough pattern i'd like a comment explaining it, according to the docs asFragment is a function so awaiting it is super odd